pkg/machines/scripts/install_machine.sh

```diff
@@ -78,7 +78,7 @@ virt-install \
     $GRAPHICS_PARAM
 EXIT_STATUS=$?
 
-if [ "$EXIT_STATUS" -eq 0 ]; then
+if [ "$EXIT_STATUS" -eq 0 ] && virsh list --all | awk  '{print $2}' | grep -q --line-regexp --fixed-strings "$VM_NAME"; then
```

Comment:
This seems to be repeating, can we wrap it in a function?
Moreover same test is performed twice in the flow (right after this one). Can it be restructured?